modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -84,7 +84,7 @@ COERCION_INDEX_BY_TYPE = {
     date: 10,
     str: 11,
     file_type: 12,   # 'file_type' is an alias for the built-in 'file' or 'io.IOBase' type.
-    object: 13,      # Any type, the 'type' attribute is not specified in a schema in a OAS document.
+    object: 13,      # The 'type' attribute is not specified in the OAS schema.
```

Comment:
@sebastien-rosset can we discuss this over Skype?
I am concerned with doing it this way as many python classes inherit from the object type so if we did it this way it would allow in a ton of other class instances that we shouldn't like set, tuple, and arbitrary other classes.
Swagger/Openapi only allows specific types (string, int, float, dict, list, bytes, datetime, date, file_type, none_type)
My preference would be to instead convert these any type classes in python into a composed schema where it oneOf allows any type. As an added benefit, this could maybe be used to verify that we can have oneOf models with simple or complex types, which I'm not sure we can do.
Things like:
oneOf:
- string
- Pet
What do you think?

Reply:
> @sebastien-rosset can we discuss this over Skype?
> I am concerned with doing it this way because many python classes inherit from the object type. So if we did it this way, it would allow in a ton of other class instances that we shouldn't like set, tuple, and arbitrary other classes.
> Swagger/Openapi only allows specific types (string, int, float, dict, list, bytes, datetime, date, file_type, none_type)
> My preference would be to instead convert these any type classes in python into a composed schema where it oneOf allows any type. As an added benefit, this could maybe be used to verify that we can have oneOf models with simple or complex types, which I'm not sure we can do.
> Things like:
> oneOf:
> 
> * string
> * Pet
> 
> What do you think?

Yes I think that makes sense. Currently, when the type is "any", the generated openapi_types() function has a dict of properties to set of allowed types, and the allowed type is "object".  That currently causes a runtime exception. So instead of having "object" in the list, we would enumerate all possible types: str, bool, date, datetime, dict, float, int, list